### DISCUSSION TITLE
screenshot in multiple-projections-instancing example

### DISCUSSION THREAD

**Discussion by emrahkavlak** [marcofugaro/threejs-modern-app#10] [Q&A]
hi marco,
thank you for creating the multi projection example! I'm a dirkkoy fan and when I found your example I got really excited.

I've got a question. When I use WebGLApp's screenshot function and change the size pixelratio to 10 (to get a large image), the resulting png is not correctly rendered. Some part disappears. Do you think this is a PNG issue or an angle / frustum / cam position issue? (FYI, if you make pixelratio 12, resulting png is 0) 

here is an example
https://staging1.eyesdontlie.co.uk/MG/dirk.html

Hope this was an ok spot to ask this question.
Thanks,
emrah

**Comment by marcofugaro** [comment]
I think you should not touch the pixel ratio, rather you should change the screenshot dimensions.
Try increasing the `width` and `height` like this:

```js
webgl.saveScreenshot({ width: 4096, height: 2160 })
```

If you're still having issues on the screenshot feature, refer to [ccapture.js](https://github.com/spite/ccapture.js/), it's what the `saveScreenshot` function is using internally.
